entities/package.py

```diff
@@ -0,0 +1,188 @@
+from typing import List
+
+from common import format_float
+
+from .common import (
+    Align, Author, BoolValue, Category, Created, Deprecated, Description, EnumValue, FloatValue, Height, Keywords,
+    Layer, Name, Polygon, Position, Rotation, Value, Version
+)
+from .helper import indent_entities
+
+
+class PackagePad():
+    def __init__(self, uuid: str, name: Name):
+        self.uuid = uuid
+        self.name = name
+
+    def __str__(self) -> str:
+        return '(pad {} {})'.format(self.uuid, self.name)
+
+
+class StrokeWidth(FloatValue):
+    def __init__(self, stroke_width: float):
+        super().__init__('stroke_width', stroke_width)
+
+
+class LetterSpacing(EnumValue):
+    AUTO = 'auto'
+
+    def get_name(self) -> str:
+        return 'letter_spacing'
+
+
+class LineSpacing(EnumValue):
+    AUTO = 'auto'
+
+    def get_name(self) -> str:
+        return 'line_spacing'
+
+
+class AutoRotate(BoolValue):
+    def __init__(self, auto_rotate: bool):
+        super().__init__('auto_rotate', auto_rotate)
+
+
+class Mirror(BoolValue):
+    def __init__(self, mirror: bool):
+        super().__init__('mirror', mirror)
+
+
+class StrokeText():
+    def __init__(self, uuid: str, layer: Layer, height: Height,
+                 stroke_width: StrokeWidth, letter_spacing: LetterSpacing,
+                 line_spacing: LineSpacing, align: Align, position: Position,
+                 rotation: Rotation, auto_rotate: AutoRotate, mirror: Mirror,
+                 value: Value):
+        self.uuid = uuid
+        self.layer = layer
+        self.height = height
+        self.stroke_width = stroke_width
+        self.letter_spacing = letter_spacing
+        self.line_spacing = line_spacing
+        self.align = align
+        self.position = position
+        self.rotation = rotation
+        self.auto_rotate = auto_rotate
+        self.mirror = mirror
+        self.value = value
+
+    def __str__(self) -> str:
+        ret = '(stroke_text {} {}\n'.format(self.uuid, self.layer) +\
+            ' {} {} {} {}\n'.format(self.height, self.stroke_width, self.letter_spacing, self.line_spacing) +\
+            ' {} {} {}\n'.format(self.align, self.position, self.rotation) +\
+            ' {} {} {}\n)'.format(self.auto_rotate, self.mirror, self.value)
+        return ret
+
+
+class Side(EnumValue):
+    THT = 'tht'
+
+    def get_name(self) -> str:
+        return 'side'
+
+
+class Shape(EnumValue):
+    ROUND = 'round'
+    RECT = 'rect'
+
+    def get_name(self) -> str:
+        return 'shape'
+
+
+class Size():
+    def __init__(self, width: float, height: float):
+        # TODO: Are width and height good names?
```

Comment:
I think those names are fine for now. Comment can probably be removed.

Reply:
Fixed.